pkg/process/config/config_nix_test.go

```diff
@@ -109,8 +109,8 @@ func TestAgentEncryptedVariablesSecrets(t *testing.T) {
 	)
 	assert.NoError(err)
 
-	assert.Equal("secret_my_api_key", config.Datadog.Get("api_key"))
+	assert.Equal("secret-my_api_key", config.Datadog.Get("api_key"))
```

Comment:
See comment below about why this is mixed with `-` and `_`, thoughts on if I should just change everything to use dashes `-`?